random.tf

```diff
@@ -0,0 +1,5 @@
+resource "random_password" "bigippassword" {
```

Comment:
Can you make this "bigip_password"?

Reply:
I've committed a change to the pull request which addresses your request